_data/codebase/v2_3/open-source/composer_lock.json

```diff
@@ -6345,7 +6345,7 @@
         },
         {
             "name": "magento/magento2-functional-testing-framework",
-            "version": "2.3.9",
+            "version": "2.3.13",
```

Comment:
do not edit codebase files manually
they can be copied from the source codebase entirely with no changes only